src/ServicePulse.Host/app/css/particular.css

```diff
@@ -1793,3 +1799,57 @@ p.col-sort-active {
     font-size: 14px;
     float: left;
 }
+
+.interactive-tooltip {
+    display: none;
+}
+
+
+.experiment:hover .interactive-tooltip {
+    display: inline;
+    position: absolute;
+    top: -34px;
+    height: 46px;
+    margin: 0 auto;
+    font-family: sans-serif;
+    text-align: center;
+    left: 50%;
+    -webkit-transform: translateX(-51%);
+    -moz-transform: translateX(-51%);
+    -ms-transform: translateX(-51%);
+    -o-transform: translateX(-51%);
+    transform: translateX(-51%);
+}
+
+.tooltip-contents {
+    text-align: center;
+    color: #fff;
+    background-color: #1A1A1A;
+    font-size: 12px;
+    padding: 6px;
+    border-radius: 4px;
+    white-space: nowrap;
+}
+
+    .tooltip-contents a {
```

Comment:
The alignment of these 2 selectors and rules is different to the rest of the rules